.gitignore

```diff
@@ -10,4 +10,6 @@ application/cache/*
 
 application/logs/*
 !application/logs/index.html
-!application/logs/.htaccess
\ No newline at end of file
+!application/logs/.htaccess
+/nbproject
```

Comment:
What is it? If it's only for your machine, should it be in your global .gitignore instead of here?